src/Tribe/Views/V2/View.php

```diff
@@ -2072,14 +2072,22 @@ static function ( &$view_data ) use ( $url_event_date, $query_args ) {
 	 * {@inheritDoc}
 	 */
 	public function url_for_query_args( $date = null, $query_args = [] ) {
-		$event_date = Dates::build_date_object( $date )->format( $this->get_url_date_format() );
-
 		if ( ! empty( $query_args ) && is_string( $query_args ) ) {
 			$str_args   = $query_args;
 			$query_args = [];
+
 			wp_parse_str( $str_args, $query_args );
 		}
 
+		// For "dateless" queries (today)
```

Comment:
* :warning: This comment is 45% valid code; is this commented out code?
* :no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: 82674694fcfc4a66a0eb745a9c39818e</sub>